Add `zcommand` for toggling fluid width mode.

whoodes opened the pull request:
**Testing Plan:** <!-- How have you tested? -->
```
tools/test-backend --coverage
tools/run-dev.py`
```

![fluid_width](https://user-images.githubusercontent.com/39782863/76599122-692c0f80-64a8-11ea-81eb-9c8605e12fc0.gif)

whoodes commented:
@showell whenever you have time I'd appreciate your input :)

timabbott commented on zerver/lib/zcommand.py:
```diff
@@ -5,36 +5,41 @@
-        return ret
-
-    night_commands = ['night', 'dark']
-    day_commands = ['day', 'light']
```
This seems to remove the `light/dark` aliases?

timabbott replied:
Oh, missed that in the commit message; OK.

timabbott commented on zerver/lib/zcommand.py:
```diff
@@ -5,36 +5,41 @@
+                                              setting='night_mode',
+                                              setting_value=True,
+                                              user_profile=user_profile))
+        return dict(msg='You are still in night mode.')
```
I'd swap the order of the cases here; it'd be more readable with the smaller thing returning first.

timabbott commented on static/js/zcommand.js:
```diff
@@ -122,6 +166,18 @@ exports.process = function (message_content) {
+        return true;
+    }
+
+    const fixed_width_commands = ['/fixed', '/standard'];
```
I don't think `/standard` is a good name to use here, since it's not clearly scoped to width.  Maybe just `fluid-width` and `fixed-width`?  With typeahead it should be fine.

whoodes commented:
@timabbott 
* Addressed your feedback regarding the return statements and command naming.
* Also refactored the `switch_setting` function as an inner function to `process_zcommands`.
* Added the typeaheads for day and night mode that were missing, as well as adding the typeaheads for fixed and fluid width.

timabbott commented:
These changes make sense to me; the whole `zcommand` system may not be something we keep as-is, but the overall effect is probably an improvement.
